mixer/pkg/lang/cel/builder.go

```diff
@@ -99,12 +99,12 @@ func NewBuilder(finder attribute.AttributeDescriptorFinder, mode LanguageMode) *
 	return &ExpressionBuilder{
 		mode:     mode,
 		provider: provider,
-		env:      env,
+		env:      *env,
```

Comment:
You'll want to change the field type to a pointer rather than doing a dereference. The previous iteration of CEL return an `interface`, whereas now the code returns a concrete `struct`